src/main/java/com/google/codeu/data/Guides.java

```diff
@@ -0,0 +1,65 @@
+package com.google.codeu.data;
+import java.util.UUID;
+
+public class Guides
+{
+	private UUID id;
+    private String name;
+    private String address;
+    private double contact_no;
```

Comment:
In Java style guide, variable names are suggested to use camelCase. So this should be: contactNo.